src/interfaces/interpreted.ts

```diff
@@ -12,7 +12,7 @@ export type Interpretation = {
     userAddress: string
     contractName: string | null
     contractAddress: string | null
-    action: Action
+    action: Action[],
```

Comment:
worth renaming this to `actions` since it's an array now.

you can use the `rename symbol` feature in vscode to do this easily